scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -538,73 +595,66 @@ EnhancedVolcano::EnhancedVolcano(deseq_results,
   theme(legend.position = "bottom")
 ```
 
-Let's make some UMAPs where we plot a gene that's identified to be differentially expressed in a given cell type and see what the expression of that gene is across samples. 
-First we need to extract the expression for each of the genes we would like to plot from the `logcounts` matrix and combine with the `colData`.
 
-```{r}
-# add column to colData with expression of gene of interest 
-# here we will actually put all of the significant genes so then we can plot any of them
-sig_expressed_genes <- deseq_results %>%
-  dplyr::filter(pvalue < 1e-5,
-                abs(log2FoldChange) > 2) %>%
-  dplyr::pull(ensembl_id)
-
-sig_genes_counts <- logcounts(rms_sce[sig_expressed_genes,]) %>%
-  as.matrix() %>%
-  t() %>%
-  as.data.frame() %>%
-  dplyr::mutate(cell_id = colnames(rms_sce))
-  
-combined_coldata_df <- as.data.frame(colData(rms_sce)) %>%
-  dplyr::left_join(sig_genes_counts, by = "cell_id")
-
-# add back modified coldata containing gene expression for sig genes 
-colData(rms_sce) <- DataFrame(combined_coldata_df, row.names = combined_coldata_df$cell_id)
-```
-
-We will pick a gene and look at the expression of that gene across subdiagnosis, sample, and cell type.
+We can also return back to the SCE object that we used to create our pseudo-bulked SCE and look at gene expression of some of the significant genes. 
+We can create UMAP plots as we did previously but instead of labeling each cell with metadata we can color each of the cells by the level of gene expression of a specified gene. 
```

Comment:
```suggestion
We can create UMAP plots as we did previously, but instead of labeling each cell with metadata, we can color cells by a specified gene's expression levels.
```